### DISCUSSION TITLE
Disable time at the end of the line

### DISCUSSION THREAD

**Discussion by eoli3n** [starship/starship#1849] [Q&A] [answered]
How to disable time at the end of the line please ?

```
~  
➜                                                                                                      11:32:02
```

**Comment by mickimnet** [comment]
That Timestamp doesn't look like it's provided by Starship.

**Comment by mickimnet** [comment, marked as answer]
For future references:
The timestamp came from a `fish_right_prompt` function.

**Comment by eoli3n** [comment]
It does, thanks
https://github.com/eoli3n/dotfiles/commit/90645a66f9d8cccd3b3b28290595cbdce031bf43#diff-6938ca68347c9fd70fa8f89709efa82aada2454492d37559e7bcee9c22aeac7f
